mibidata/util.py

```diff
@@ -110,3 +111,23 @@ def format_for_filename(label):
     for char in set(label).intersection(DELIMITERS):
         label = label.replace(char, '-')
     return label
+
+
+def car2pol(x, y, x_c=0, y_c=0):
+    """Convert cartesian to polar coordinates w.r.t. a central point.
+    Angle phi is returned in deg in the range [0, 360) deg.
```

Comment:
Is there a reason to do this in degrees and not radians?

Reply:
I find it more user friendly to use degrees instead of radians, but I can change it if you prefer radians.

Reply:
done